renovate.json5

```diff
@@ -0,0 +1,25 @@
+{
+  // start with default Renovate settings
+  "extends": [
+    "config:base"
+  ],
+
+  // rules for specific subsets of packages
+  "packageRules": [
+    // for everything except major updates
+    {
+      "updateTypes": ["minor", "patch", "pin", "digest", "lockFileMaintenance"],
+
+      // automatically merge, as long as tests pass
+      "automerge": true
+    }
```

Comment:
This will automatically merge packages that pass tests, as long as they're not major version updates. I think we want to manually approve these.